gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/apim/core/scoring/domain_service/ScoreComputingDomainService.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright © 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.apim.core.scoring.domain_service;
+
+import io.gravitee.apim.core.DomainService;
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+
+@DomainService
+public class ScoreComputingDomainService {
+
+    public Double computeScore(Long nbErrors, Long nbWarnings, Long nbInfos, Long nbHints) {
```

Comment:
you don't accept null value and you return always a value, we can use type to say to use that don't accept null